src/store/auth/index.js

```diff
@@ -1,18 +1,35 @@
-export default {
+import { auth } from 'firebase'
+import { firebaseMutations } from 'vuexfire'
+
+export default{
   namespaced: true,
   state: {
     // Add auth related state
+    user: null,
+    isAdmin: false
   },
 
   getters: {
     // Add auth related getters
+    user: state => state.user
```

Comment:
If a getter is just returning state, it's not needed. I may have made mistakes in some past projects

Reply:
And I proudly carried those mistakes forward :sweat_smile: